codeforboston/maple
# 946 extract test from learn testimony components

# Summary

Extracted strings from Learn Testimony page and make it translate-able
# Checklist


# Screenshots
Extracted texts
<img width="1012" alt="Screenshot 2024-07-31 at 8 57 25 AM" src="https://github.com/user-attachments/assets/828ca6a1-dac8-49bc-90e6-21cfaf98d81d">
anges._

# Steps to test/reproduce

1. Go to the main website page.
2. Access the Navigation Menu in the top-left corner
3. Locate the "Learn" Section
4. Check if everything in the learn pages are working well
